Realm/Tests/SwiftUISyncTestHost/ContentView.swift

```diff
@@ -0,0 +1,310 @@
+                        Button("Navigate Next View") {
+                            canNavigate = true
+                        }
+                        .accessibilityIdentifier("show-list-button-view")
+                    }
+                }
+            case .error(let error):
+                ErrorView(error: error)
+                    .background(Color.red)
+                    .transition(AnyTransition.move(edge: .trailing)).animation(.default)
+            case .progress(let progress):
+                ProgressView(progress)
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+                    .background(Color.yellow)
+                    .transition(AnyTransition.move(edge: .trailing)).animation(.default)
+                    .onAppear {
+                        self.progress = progress
+                    }
+            }
+        }
+    }
+}
+
+struct AutoOpenView: View {
+    @State var canNavigate: Bool = false
+    @State var progress: Progress?
+    @AutoOpen(appId: ProcessInfo.processInfo.environment["app_id"]!,
+              partitionValue: ProcessInfo.processInfo.environment["function_name"]!,
+              timeout: 2000) var autoOpen
+
+    var body: some View {
+        VStack {
+            switch autoOpen {
+            case .connecting:
+                ProgressView()
+            case .waitingForUser:
+                ProgressView("Waiting for user to logged in...")
+            case .open(let realm):
+                if canNavigate {
+                    ListView()
+                        .environment(\.realm, realm)
+                        .transition(AnyTransition.move(edge: .leading)).animation(.default)
+                } else {
+                    VStack {
+                        Text(String(progress!.completedUnitCount))
+                            .accessibilityIdentifier("progress-text-view")
+                        Button("Navigate Next View") {
+                            canNavigate = true
+                        }
+                        .accessibilityIdentifier("show-list-button-view")
+                    }
+                }
+            case .error(let error):
+                ErrorView(error: error)
+                    .background(Color.red)
+                    .transition(AnyTransition.move(edge: .trailing)).animation(.default)
+            case .progress(let progress):
+                ProgressView(progress)
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+                    .background(Color.yellow)
+                    .transition(AnyTransition.move(edge: .trailing)).animation(.default)
+                    .onAppear {
+                        self.progress = progress
+                    }
+            }
+        }
+    }
+}
+
+struct AsyncOpenPartitionView: View {
+    @AsyncOpen(appId: ProcessInfo.processInfo.environment["app_id"]!,
+               partitionValue: "wrong_partition_value",
+               timeout: 2000) var asyncOpen
```

Comment:
`var` should be at the start of a new line when the property wrapper is multiple lines.